### PR TITLE
Feature/migrate mask data to native

### PR DESCRIPTION
**Pull request by qdsfdhvh**
Jira MM-298 MM-299 MM-300 MM-301 MM-57

### CODE REVIEW

**Review by Tlaster — APPROVED** [review]
LGTM!

File: persona/src/androidMain/kotlin/com/dimension/maskbook/persona/datasource/DbPersonaDataSource.kt [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import kotlinx.coroutines.flow.map
+
+class DbPersonaDataSource(database: PersonaDatabase) {
+
+    private val personaDao = database.personaDao()
+    private val linkedProfileDao = database.linkedProfileDao()
+    private val relationDao = database.relationDao()
+
+    suspend fun deletePersona(personaIdentifier: String) {
+        personaDao.delete(personaIdentifier)
```
Comment (JoeSteven) [inline persona/src/androidMain/kotlin/com/dimension/maskbook/persona/datasource/DbPersonaDataSource.kt]: Maybe it's better to wrap delete code with database.withTransactions

**Review by JoeSteven — APPROVED** [review]
LGTM!

**Review by HuixingWong — APPROVED** [review]
LGTM！

**Pull request merged**